napi.h

```diff
@@ -142,6 +142,26 @@ static_assert(sizeof(char16_t) == sizeof(wchar_t),
     }                                                                          \
   } while (0)
 
+#define NAPI_INTERNAL_STRINGIFY_(x) #x
+#define NAPI_INTERNAL_STRINGIFY(x) NAPI_INTERNAL_STRINGIFY_(x)
+
+// Internal check helper. Be careful that the formatted message length should be
+// max 255 size and null terminated.
+#define NAPI_INTERNAL_CHECK(expr, location, ...)                               \
+  do {                                                                         \
+    if (!(expr)) {                                                             \
+      std::string msg = Napi::details::SPrintF(__VA_ARGS__);                   \
+      Napi::Error::Fatal(location " " #expr, msg.data());                      \
```

Comment:
Consider using `c_str()` instead of `data()`.

Reply:
Thanks, fixed